### DISCUSSION TITLE
Wallpaper does not pause when fullscreening other programs when stretching the same wallpaper to multiple displays.

### DISCUSSION THREAD

**Discussion by turffff** [rocksdanister/lively#1796] [General]
![image](https://github.com/rocksdanister/lively/assets/139887637/4c8a32be-0e38-4add-8bb7-f3e1761bb6ae)
(task manager showing that the background isn't paused)

![image](https://github.com/rocksdanister/lively/assets/139887637/f7d1e28a-863f-4f26-839a-90eae02a204e)
(task manager showing that the background is paused when set to one wallpaper per screen)

The pausing feature for when you fullscreen an app works like it should when using one wallpaper per screen, but when using the stretch wallpaper mode, it doesn't pause, causing unnecessary GPU usage.

![image](https://github.com/rocksdanister/lively/assets/139887637/271b974a-0192-4118-9dce-fc1eaa4c3300)
(just proof that i actually have enabled pausing)

**Comment by rocksdanister** [comment]
If you want wallpaper to pause if any of the screen is filled in span set `Display pause` rule to `All screen(s)`
![image](https://github.com/rocksdanister/lively/assets/17554161/4fc11807-0535-40e6-be9d-424fcdbd591a)

**Reply by turffff** [reply]
No, the issue is that the background behind the monitor i'm using is not paused, If you use the settings you show here, the background stops moving on monitors it "should" be moving in (like doing something only on one screen for example and having the bg move on the other monitors with nothing in fullscreen). 

Ultimately the ideal would be that the stretched background would pause per screen instead of being one big entity

**Reply by rocksdanister** [reply]
I dont understand what you mean, stretch just means one wallpaper stretched to all screens.. since there is only one wallpaper running what else can be done besides pausing on all screens.

**Reply by turffff** [reply]
hmmm, i see the problem, sorry for the bother.
